solr_config/schema.xml

```diff
@@ -180,18 +179,16 @@
   <field name="merged_rsID" type="text_general" indexed="true" stored="true" multiValued="false" />
 
   <!-- specific fields for Gene -->
-  <!-- <field name="reported_trait" type="text_general" indexed="true" stored="true" multiValued="false" /> -->
-  <field name="geneDescription" type="text_general" indexed="false" stored="true" multiValued="false" />
-  <field name="ensemblID" type="string" indexed="true" stored="true" multiValued="false" />
-  <field name="geneName" type="string" indexed="true" stored="true" multiValued="false" />
-  <field name="synonymsGene" type="text_general" indexed="true" stored="true" multiValued="false" />  
+  <field name="ensemblDescription" type="string" indexed="true" stored="true" multiValued="false" />
+  <field name="entrez_id" type="string" indexed="true" stored="true" multiValued="false" />
+  <field name="cytobands" type="text_general" indexed="true" stored="true" multiValued="false" />  
   <field name="biotype" type="text_general" indexed="true" stored="true" multiValued="false" />
-  <field name="variants" type="string" indexed="true" stored="true" multiValued="true" />
-
-  <field name="mapping" type="text_general" indexed="true" stored="true" multiValued="true" />  
-  <field name="mapping.chr" type="text_general" indexed="true" stored="true" multiValued="false" />  
-  <field name="mapping.start" type="int" indexed="true" stored="true" multiValued="false" />  
-  <field name="mapping.end" type="int" indexed="true" stored="true" multiValued="false" />  
+  <field name="rsIDs" type="string" indexed="true" stored="true" multiValued="true" />
+  <field name="synonymsGene" type="text_general" indexed="true" stored="true" multiValued="false" />
+  <field name="crossRefs" type="text_general" indexed="true" stored="true" multiValued="false" />
```

Comment:
Is the expectation that users would search on "crossRefs"? If no, then this indexed can be set to false.

Reply:
It's sometimes happen that users search for Entrez IDs or refseq IDs.

Reply:
Ok, I thought in another conversation it was mentioned this doesn't occur, which is why I was asking. Looks good then!